app/artifact-cas/internal/server/grpc.go

```diff
@@ -93,6 +94,18 @@ func NewGRPCServer(c *conf.Server, authConf *conf.Auth, byteService *service.Byt
 	if c.Grpc.Timeout != nil {
 		opts = append(opts, grpc.Timeout(c.Grpc.Timeout.AsDuration()))
 	}
+	cert := c.Grpc.TlsConfig.Certificate
+	privKey := c.Grpc.TlsConfig.PrivateKey
+	if cert != "" && privKey != "" {
+		cert, err := tls.LoadX509KeyPair(cert, privKey)
+		if err != nil {
+			return nil, err
+		}
+		opts = append(opts, grpc.TLSConfig(&tls.Config{
+			Certificates: []tls.Certificate{cert},
+			MinVersion:   tls.VersionTLS12,
```

Comment:
out of curiosity, why do you set this? It the default insecure? If so, it might make sense to have a comment.

Reply:
Yes, when not set, gosec complains about using a insecure minimum version.